.eslintignore

```diff
@@ -1,4 +1,5 @@
 *.min.*
+node_modules
```

Comment:
added this as there are still some unhoisted dependencies (`moment`) that will be installed in `markbind` (lib)